index.js

```diff
@@ -444,34 +466,30 @@ async function materializeModifiedSchemas(gitRoot = undefined, options = {}) {
  * @return {Object} {title, uri, version, current<boolean>, schema<Object>}
  */
 async function schemaPathToInfo(schemaPath, options = {}) {
-    _.defaults(options, defaultOptions);
-
-    const schema = await readObject(schemaPath);
-
-    return {
-        title: _.get(schema, options.schemaTitleField, null),
-        path: schemaPath,
-        version: schemaVersion(schema, options.schemaVersionField),
-        current: path.parse(schemaPath).base === options.currentName,
-        schema,
-    };
+    return _schemaPathToInfo(
+        await readObject(schemaPath),
+        schemaPath,
+        options
+    );
 }
 
 /**
- * Looks in schemaBasePath for files that look like schema files.
- * These are either X.Y.Z.<contentType> files or currentName.<contentType>
- * files.
- * @param {string} schemaBasePath
+ * Synchronous version of schemaPathToInfo
+ * @param {string} schemaPath
  * @param {Object} options
- * @return {Array}
+ * @return {Object} {title, uri, version, current<boolean>, schema<Object>}
  */
-async function findSchemaPaths(schemaBasePath, options = {}) {
+function schemaPathToInfoSync(schemaPath, options = {}) {
+    return _schemaPathToInfo(readObjectSync(schemaPath), schemaPath, options);
+}
+
+function _findSchemaPaths(dirContent, schemaBasePath, options = {}) {
```

Comment:
Since this isn't doing any 'finding' anymore, let's rename this to _filterSchemaPaths

Reply:
Not relevant anymore.